src/apps/menu/menu.test.ts

```diff
@@ -1,9 +1,10 @@
-import { TOKEN_LIBRARY_KEY } from "../../core/token";
+import { TOKEN_LIBRARY_KEY, TOKEN_USER_KEY } from "../../core/token";
 
-describe("Menu", () => {
+describe("Menu (authenticated))", () => {
   beforeEach(() => {
     cy.window().then((win) => {
       win.sessionStorage.setItem(TOKEN_LIBRARY_KEY, "random-token");
+      win.sessionStorage.setItem(TOKEN_USER_KEY, "random-user");
```

Comment:
I think you should use `cy.createFakeAuthenticatedSession()` instead.
The reason why it was made was that then we don't need to know what is happening in order to create a fake user session. It might even change or be extended in the future, and then we only have to change it in that function.